client/src/utils/theme.js

```diff
@@ -0,0 +1,17 @@
+export const light = {
+  theme: "light",
+  bg: "#ededed",
+  color: "#707070",
+  hcolor: "#0f4e8f",
+  detailCard: "white",
+  a: "#707070",
+};
+
+export const dark = {
+  theme: "dark",
+  bg: "#3d4849",
+  color: "#ffffff",
+  hcolor: "#ffffff",
+  detailCard: "#535F60",
+  a: "white",
+};
```

Comment:
I would stick to the same pattern when it comes to naming your props. So `color` could be `p` (or `text` if you use it for `span`, too), `hcolor` could simply be `h` and `detailCard` should probably be `bg-detailCard`. Maybe this makes it easier to read and to recall later :)